src/calculateRentalCost.js

```diff
@@ -4,7 +4,34 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+  const LOW_TERM = 3;
+  const LOW_TERM_DISCONT = 20;
+  const price = days * COST_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return price - LONG_TERM_DISCONT;
+  }
+
+  if (days >= LOW_TERM) {
+    return price - LOW_TERM_DISCONT;
+  }
+
+  return price;
+
+  // if (days < LOW_TERM) {
+  //   price = days * COST_PER_DAY;
+  // } else if (days >= LOW_TERM) {
+  //   price = days * COST_PER_DAY - LOW_TERM_DISCONT;
+  // }
+
+  // if (days >= LONG_TERM) {
+  //   price = days * COST_PER_DAY - LONG_TERM_DISCONT;
+  // }
+
+  // return price;
```

Comment:
The commented-out code is redundant and can be removed to improve readability. The logic for calculating the price is already implemented above and does not require these additional comments.